tools/run_tests.sh

```diff
@@ -27,6 +27,27 @@ echo_client_path="${TCF_HOME}/examples/apps/echo/client"
 generic_client_path="${TCF_HOME}/examples/apps/generic_client"
 # Read Listener port from config file
 listener_port=`grep listener_port ${TCF_HOME}/config/tcs_config.toml | awk {'print $3'}`
+LISTENER_URL="localhost"
+
+while getopts "l:lh" OPTCHAR ; do
+    case $OPTCHAR in
+        l )
+            LISTENER_URL=$OPTARG
+            ;;
+        \?|h )
+            BN=$(basename $0)
+            echo "$BN: Run tests for Hyperledger Avalon" 1>&2
+            echo "Usage: $BN [-l|-h|-?]" 1>&2
+            echo "Where:" 1>&2
+            echo "   -l       specify the Listener service name" 1>&2
+            echo "   -? or -h print usage information" 1>&2
+            echo "Examples:" 1>&2
+            echo "   $BN -l" 1>&2
```

Comment:
This example is not correct.  Use Docker usage as the example:

`echo "   $BN -l avalon-listener" 1>&2`

Reply:
ok